cmd/risor-docs/go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/risor-io/risor/cmd/risor-docs
 
-go 1.21.4
+go 1.21
```

Comment:
For some reason, and even though I always run the latest version, `gopls` fails when the version number includes the patch version and there's a `replace` statement in `go.mod`, there are some reports of this issue upstream but it hasn't been fixed yet, so if you don't mind, I'm dropping the patch version here.